src/main/python/covata/delta/utils.py

```diff
@@ -59,3 +30,9 @@ def _f(*args, **kwargs):
             return function(*args, **kwargs)
         return _f
     return decorator
+
+
+def check_id(arguments):
+    return check_arguments(arguments,
```

Comment:
I'm more inclined to not enforce the id check, because it's something that is generated by the server, and it's hard to say whether there's any added value of having that check in..